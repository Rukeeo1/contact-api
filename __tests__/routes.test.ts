import request from 'supertest';
//import app from '../src/routes/api';
import app from '../src/app';

// describe('Api Routes', () => {
//   test('/home returns hello world', () => {
//     return request(app)
//       .get('/api/home')
//       .expect(200, { data: 'Hello World' });
//   });
// });

describe('api routers', () => {
  it('shoul return everything in the data base', () => {
    return request(app)
      .get('/api/contacts')
      .expect(200, [
        {
          id: '1',
          name: 'pulling chichi out booishzyn',
          email: 'yyyyx@gmail.com',
          phone: '0818-384-0096',
          company: 'we are themo',
          isBlocked: 'false'
        },
        {
          id: '2',
          name: 'Ai Obi',
          email: 'anitachiz@gmail.com',
          phone: '0818-000-0000',
          company: 'anita and sons',
          isBlocked: 'false'
        },
        {
          id: '3',
          name: 'Johnson Okoro',
          email: 'okorojohnson@gmail.com',
          phone: '0818-001-0001',
          company: 'okoro and sons',
          isBlocked: 'false'
        },
        {
          id: 'e1872523-bd32-4400-a3f3-b09fbf347c44',
          name: 'hello booishzyn',
          email: 'yyyyx@gmail.com',
          mobile: '0818-384-0096',
          isBlocked: 'false',
          company: 'we are themo'
        },
        {
          id: '97b5667f-33f0-44a3-a6e8-4b75a88e7b0c',
          name: 'rukee ojigbo',
          email: 'yyyyx@gmail.com',
          mobile: '0818-384-0096',
          isBlocked: false
        },
        {
          id: '0795bf27-3d99-4b0c-999d-3189ced97a19',
          name: 'ochuko ojigbo',
          email: 'ochuko@gmail.com',
          mobile: '0818-384-0096',
          isBlocked: false
        },
        {
          id: '118136af-98af-4752-a3fa-7e5560d6ee45',
          name: 'Linda Ikeji',
          email: 'lindaikeji@gmail.com',
          mobile: '0818-384-0096',
          isBlocked: false
        }
      ]);
  });
});

describe('it should return hello world', () => {
  it('should return hello world', () =>{
return request(app).get('/dummy').expect('i worked')
  })
})