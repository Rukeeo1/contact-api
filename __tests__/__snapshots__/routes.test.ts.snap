// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GET ALL CONTACTS should return everything in the data base: contacts 1`] = `
Array [
  Object {
    "company": "we are themo",
    "email": "yyyyx@gmail.com",
    "id": "1",
    "isBlocked": "false",
    "name": "pulling chichi out booishzyn",
    "phone": "0818-384-0096",
  },
  Object {
    "company": "anita and sons",
    "email": "anitachiz@gmail.com",
    "id": "2",
    "isBlocked": "false",
    "name": "Ai Obi",
    "phone": "0818-000-0000",
  },
  Object {
    "company": "okoro and sons",
    "email": "okorojohnson@gmail.com",
    "id": "3",
    "isBlocked": "false",
    "name": "Johnson Okoro",
    "phone": "0818-001-0001",
  },
  Object {
    "company": "we are themo",
    "email": "yyyyx@gmail.com",
    "id": "e1872523-bd32-4400-a3f3-b09fbf347c44",
    "isBlocked": "false",
    "mobile": "0818-384-0096",
    "name": "hello booishzyn",
  },
  Object {
    "email": "yyyyx@gmail.com",
    "id": "97b5667f-33f0-44a3-a6e8-4b75a88e7b0c",
    "isBlocked": false,
    "mobile": "0818-384-0096",
    "name": "rukee ojigbo",
  },
  Object {
    "email": "ochuko@gmail.com",
    "id": "0795bf27-3d99-4b0c-999d-3189ced97a19",
    "isBlocked": false,
    "mobile": "0818-384-0096",
    "name": "ochuko ojigbo",
  },
  Object {
    "email": "lindaikeji@gmail.com",
    "id": "118136af-98af-4752-a3fa-7e5560d6ee45",
    "isBlocked": false,
    "mobile": "0818-384-0096",
    "name": "Linda Ikeji",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "95cd22fc-b8fd-4ee2-b18c-d16608240bdb",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "5b6aa79f-d3d6-4112-bd0d-6a540c498245",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "4e697ae0-4a3c-4aa5-879e-170732342660",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "8424e3ca-81d1-4451-9684-d1fd4555da55",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "d6bf34ab-2587-4006-9c8d-592aed347e50",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "af39ae81-8978-4741-a501-e8661857269a",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "98a8b800-a9f9-4a29-bf4d-bd04c4970f12",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "99120457-72a5-4d29-b681-9d07ac321f8f",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "5847acc8-8354-462c-9049-8366cd13c13b",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "455e2173-47df-4b08-95cc-9f6e49220a6f",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "a201025b-f84e-4795-bf9e-74acd448bb00",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "8576d12b-f936-4d0d-ab8c-5c409c244102",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "982f4616-ef87-4c44-9257-7b9f1594c82f",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "878f47da-b0c1-4bb3-af5d-4e53ea2c8fca",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "f987bc9b-0e2f-4242-a64e-7fd4010deb09",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "7f2797ea-0338-418f-9beb-6c7eb294cc3d",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "cc2931d0-ef56-4818-abed-336e77d8b8da",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "849450fa-1cde-40d4-9524-59888953bc99",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "3d160e41-9770-4155-b052-3f05138c6902",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "cd604ff9-6c04-4efc-92bb-9e1281700bd8",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "799c508d-2f32-45d6-82a1-795d380e7e89",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "cf93eb55-61eb-47be-b19a-a263a659ce81",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "be641af8-7173-4762-8c13-29402e9d80a1",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "b26b975d-2396-48c4-be13-a0b6a994ad0d",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "9ef8a882-24f3-4414-8cb9-c069725e3381",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "a1f26148-ccbc-4b44-86eb-5d45d7746810",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "a22665ce-eae1-4064-a825-ed4cc25e69ae",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "3ecdb06e-fc75-4cec-8ff0-1819b0691220",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "6ad2041d-0d85-432d-a75f-0d6b5a9a99cd",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "ed8521d0-be22-4dea-98b8-2a024fe50f12",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "208aef3c-a808-451e-865a-3a3064b7364e",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "792e7fc9-c996-4954-a968-4fac2ba9ea0f",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "2c622760-7026-4cc4-9987-1af06348cbf9",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "5aee3f80-6a6c-46e2-83d3-c268fbea6fc9",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "af7393bf-3459-435d-bcad-ae045426fc5b",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "ad9b1914-941f-4a53-bc12-a475a9f25ab8",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "5a0e9a18-4637-4ec6-bde3-585e16b949f5",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "9605a02a-2f4f-408f-b750-ce38945f574d",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "839b0c1f-0125-4e9b-99aa-79523ecdc24b",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "3b459cf5-7e95-4281-beb8-55172e7bd6bf",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "fa2510b9-a9f3-4ed5-86f4-9cd8cc391fbf",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "c112dccc-7725-4ebb-9a41-a76d610ea6ad",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "23b27ffa-333a-48ee-b1c5-5954ab3a5c70",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "90f91851-c6f7-48a8-97b9-7d71cfde06b8",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "516856d9-c95b-4156-a212-630365e59936",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "03545a13-fbce-4a50-9331-c4f2109a0581",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "0b286a74-5992-487c-bebb-150fbf5e7c96",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "c86808ca-fd16-499c-8767-0affc2429a38",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "a53bf6ac-8af4-4be9-bcff-6376fede0473",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "26b9cbf8-fba7-4fe6-8db4-4a60e1ebf50f",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "e88f41ed-002c-41e8-871d-b404d703ec11",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "d84750ac-4913-4b24-ba45-7ed1f71c6743",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "c1d389fb-bc70-47d6-b784-c954154e3a0c",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "995242a7-4ed1-4518-9b69-ec3641bd08b2",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "deda03d9-b9e7-42d1-9eb9-b92a61164939",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "93cc812e-84ba-4319-b0c9-053f32cf4bfa",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "a7c5510e-7ec8-4ab0-9c6f-bc3cc70cfb1c",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "8e11b547-b0fa-455f-952b-b61c90e4bf91",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "666f75c4-3c7c-4dc9-b452-ff3ea7267bf1",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "e586dd55-6219-4324-b4eb-5a739e1be7d5",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "923ff00d-da65-4eeb-a9c1-f11184849b1d",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "9ea1b12c-a5eb-416d-8597-894487afd4d1",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "0ae20422-dd86-4c6c-aea9-8b5225000bdb",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "f94d081f-a466-4a4e-9b9e-031f58bb89c7",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "b2059ead-0e2e-4f60-b5f7-b33716075b86",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "4fd76cee-598b-4374-8670-1d257e98cfb6",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "68bd7598-2fd2-493a-97d1-d64fddb8d8c7",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "b079824c-502f-44cd-b0d9-68d0e3da5789",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "0781da04-5cf9-4125-ace3-8af23a552c67",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "b0417e0c-c3b6-4ae5-bf85-0b66d39e4254",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "052dcd1a-fc41-4dfd-85d8-f2f8d52a2b5b",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "dee2061b-e204-43e5-aa28-5a745e2c866c",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "ce35bd5f-6f49-4068-835b-cb498362af35",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "a8359472-e65f-44f7-bf92-971e7252f47a",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "e2196c82-3ba5-4e9f-86ce-95ef769efaf7",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "edeeaf88-db23-4c64-bb77-c3219470b1ce",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "124b8cf0-6f18-48f9-b7e2-b668c38d41a4",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "cf8dd15b-b62f-444e-ae86-eeac56fac265",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "4a25d631-ca85-4bee-ac2f-aba47e62fddc",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "db358989-4eb9-4294-ad98-5f9851de0453",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "c3e3623f-7907-4675-acb4-ba77c5595ac1",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "7ef50fda-ae49-481e-8d90-8ac25af68f1d",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "c338b1f9-10bc-4aed-b6ee-0d9e64626201",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "6b35c907-dfc2-4720-b853-13fb638b17ec",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "9acc8a88-5eba-44bf-a371-4057921ce144",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "10b87c36-7aa5-484d-830b-74ce2cfa32db",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "e4954f5a-c75b-4dc6-ba67-373ad4315325",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "b3e396b2-723c-454b-ba68-c24932543ebd",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "89a4e617-1b0e-4091-a6e6-75c7a865d9d2",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "bd1d15b6-d2db-4714-9f6a-082791b1a1ef",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "fabe7b13-adad-450b-aaa4-030767c7e21a",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "d57088c5-568b-416a-ade0-324ba406c525",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "be13bf5a-f878-4d2f-934f-b98aa084d10b",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "b5d06b79-f6f9-4603-bae6-97b828921c78",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "9692d456-f61d-43b9-8788-87f29c9ff1be",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "e987df60-895b-4e61-801f-cec249b2bdc2",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "b726f230-5e9b-4746-a5e5-9179386e64db",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
  Object {
    "email": "sharon@gmail.com",
    "id": "24652079-a47c-4b93-af1d-494db16bb9f6",
    "isBlocked": false,
    "mobile": "0803-668-7418",
    "name": "Sharon Stone",
  },
]
`;
